gluten-core/src/main/java/org/apache/gluten/vectorized/JniLibLoader.java

```diff
@@ -72,7 +62,7 @@ public class JniLibLoader {
   private final Set<String> loadedLibraries = new HashSet<>();
   private final Lock sync = new ReentrantLock();
 
-  JniLibLoader(String workDir) {
+  public JniLibLoader(String workDir) {
```

Comment:
Can we keep the ctor package-private? If no other places are calling it.

Reply:
Done thanks!